test/etaoin/api_test.clj

```diff
@@ -18,11 +18,26 @@
   (or (instance? Double val)
       (instance? Integer val)))
 
+; By default we run the tests with all the drivers supported on the current OS.
+; To override this, you can set the environment variable ETAOIN_TEST_DRIVERS
+; to a Clojure vector encoded as a string; for example:
+;
+;   ETAOIN_TEST_DRIVERS="[:firefox]" lein test
+;
+(def drivers
```

Comment:
this declaration looks a bit clumsy. Would be great to have something like 

```
(or (get-drivers-from-env)
    (get-drivers-from-prop)
   (def-default-drivers
```

to make the logic a bit cleaner. You may improve it later.